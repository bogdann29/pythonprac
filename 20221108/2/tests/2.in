gas
1, 1, 2, 2, 1, 2
[0, 0], [0, 0], [0, 0]
(0, 0), (1, 1), (2, 0)
asg
(1, 5), (10, 15), (11, 14)