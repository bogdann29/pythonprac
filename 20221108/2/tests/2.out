Invalid Input
Invalid Input
Not a triangle
1.00
